Makefile

```diff
@@ -130,13 +181,30 @@ endif
 	@sed -i 's,newTag: .*,newTag: $(NOTEBOOK_TAG),g' $(NOTEBOOK_DIR)/kustomization.yaml
 	$(KUBECTL_BIN) apply -k $(NOTEBOOK_DIR)
 
+.PHONY: deploy9
+deploy9-%-ubi9-python-3.9: bin/kubectl
+	$(eval NOTEBOOK_DIR := $(subst -,/,$(subst cuda-,,$*))/ubi9-python-3.9/kustomize/base)
+ifndef NOTEBOOK_TAG
+	$(eval NOTEBOOK_TAG := $*-ubi9-python-3.9-$(IMAGE_TAG))
```

Comment:
This needs to be updated to include `generate-image-tag`, similar to https://github.com/opendatahub-io/notebooks/pull/30/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R127, else it will use only the `IMAGE_TAG` value.

Alternatively, we can call `generate-image-tag` when we first define [IMAGE_TAG](https://github.com/opendatahub-io/notebooks/pull/30/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R3) variable